src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -550,6 +561,55 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			// is this request the full path to a real file?
+			if (File.Exists(localPath) && Path.IsPathRooted(localPath))
+			{
+
+				// currently this only applies to languageDisplay.css, settingsCollectionStyles.css, and customCollectionStyles.css
+				var cssFile = Path.GetFileName(localPath);
+				if ((cssFile == "languageDisplay.css") || (cssFile == "settingsCollectionStyles.css") || (cssFile == "customCollectionStyles.css"))
+				{
+					info.ContentType = "text/css";
+					info.ReplyWithFileContent(localPath);
+					return true;
+				}
+			}
+
+			// if not a full path, try to find the correct file
+			var fileName = localPath;
+			var pos = fileName.LastIndexOfAny(new[] { '\\', '/' });
+			if (pos > -1)
+				fileName = fileName.Substring(pos + 1);
+
+			// try to find the css file in the xmatter and templates
+			var path = (_fileLocator == null) ? string.Empty : _fileLocator.LocateFile(fileName);
+
+			// try to find the css file in the BloomBrowserUI directory
+			if (string.IsNullOrEmpty(path))
+			{
+				// collect the css files in the BloomBrowserUI directory
+				if (_bloomBrowserUiCssFiles == null)
+				{
+					var sourceDir = FileLocator.GetDirectoryDistributedWithApplication("BloomBrowserUI");
+					_bloomBrowserUiCssFiles = Directory.EnumerateFiles(sourceDir, "*.css", SearchOption.AllDirectories).ToArray();
+				}
```

Comment:
So now we're looking for css files anywhere in BloomBrowserUI, not just in the places the file locator would already find them there? Seems to me that if we're missing some it might be easier to just add the missing directories to the file locator...